### ISSUE TITLE
FTP instead of SFTP

### ISSUE THREAD

**Issue by dillonplunkett** [DarwinBaker/AATool#8]
This is an amazing tool. Thanks for making it! I'm trying to use it for co-op on a dedicated server for the first time, but discovered that the hosting that we're using provides FTP access and not SFTP access. Is there a straightforward way to use it with only FTP access?

**Comment by DarwinBaker** [comment]
Hey, glad to hear you like it! Unfortunately SFTP is a completely different protocol under the hood (SFTP uses a secure channel), so the two aren't compatible. I could program support for it, but I would need somebody to lend me their server login to test against. 

That being said, I believe you should still be able to set it up outside of the tracker. If you can mount the FTP folder of the server as a network drive/folder in Windows, you can just set your tracker to use that drive letter/folder path.

I believe you can do this in Windows by right clicking on "This PC" in the left browser pane of the file explorer then clicking "Map network drive..." Then click "Connect to a Web site that you can use to store your documents and pictures"
![image](https://user-images.githubusercontent.com/51175243/133161196-122cb64b-0a49-4bff-b94b-793a75ca8107.png)

I haven't tried this myself since I don't have a server, but I have no reason to believe this shouldn't work. The only thing about doing it this way would be that you won't get the "next refresh estimate" feature that tells you how long until the next server autosave. Hope this helps, and good luck on the runs!

**Comment by dillonplunkett** [comment]
Thanks so much for the detailed reply! I gave this method a try and was able to add the ftp connection to the server, but it looks like it's added as a network location, not mounted as a drive. So I can't choose it in the Browse for Folder menu within AATool. Based on some quick searching, I think it can be mounted in a way that would work using third-party tools (haven't tried any yet), but not natively in Windows. 

If you're interested in adding ftp support and just need ftp access to a server to test against, I'd be happy to get another server through the hosting service we're using. It's only a few dollars. Just let me know and I'll set it up and send you credentials.

Thanks again for your help!

**Comment by dillonplunkett** [comment]
Yup, definitely still happy to help. Just let me know what email address is best and I'll set up a server and send you credentials. Or if you don't want to post an email address publicly, you can reach me at this username at gmail.

**Comment by dillonplunkett** [comment]
Just wanted to check in on this. I totally understand if you're just too busy or if other features are more pressing, but if you're still interested in adding ftp support, I'm still happy to pay for a server for a month.

**Comment by mbanozic** [comment]
Most hosting websites only offer FTP in their plans, I'd also like to see FTP support...

**Comment by Zunwolf1896** [comment]
Also very interested in FTP support, any update on this?

**Comment by mcmonkey819** [comment]
Just wanted to add a +1 to this feature request and a note about how I worked around the lack of FTP. There's a freeware tool called FTPUse that can be used to mount an FTP folder as a drive in Windows. I followed the instructions on this page and setup a batch file to run on startup that mounts the drive and things now work flawlessly. [Map FTP as Drive](https://www.wintips.org/map-ftp-to-local-drive-letter-windows/#part-1https://www.ferrobackup.com/map-ftp-as-disk.html)

**Comment by SnipyPlays** [comment]
Is this still a possibility in the future? I would love to use this on my servers, but the option given is not working for me.

**Comment by dillonplunkett** [comment]
@SnipyPlays I haven't been able to get FTPUse to work fluidly, but [DriveMaker](https://backupchain.com/en/drivemaker/) seems to work well. However, it's not free software (or rather the free version has significant limitations). 

@DarwinBaker I'm still happy to cover the cost of a server for you (or set one up for you) for a month if you're interested in adding native FTP support.
